tsconfig.test.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "ES2022",
+    "target": "ESNext",
+    "moduleResolution": "Node",
+    "esModuleInterop": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "sourceMap": true
+  },
+  "include": ["src/**/*.ts", "src/tests/*.test.ts"],
```

Comment:
Os testes estarão no próprio diretório do módulo/componente/página, não?
Se for o caso, esse caminho está incorreto.